app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -164,229 +167,118 @@ For the following Ukrainian and Afghan visas, you can apply for your visa from w
 
-If you are applying to train to teach French, German or Spanish, your bachelor’s degree must meet the standard of a UK degree class 2.1 or above and be one of the following:  
+You will be eligible for the IRP. 
 
-* a French, German or Spanish degree 
+#### Example 3:  
 
-* a degree combining French, German or Spanish 
+You study in the UK on a Student visa for three years from September 2019 to July 2022, before returning home. You apply for an eligible teaching job in England starting in September 2024. You apply for a Skilled Worker visa in June 2024, 3 months before the start of your job. You move to the UK in July 2024.  
 
-* a degree in any subject, taught entirely in French, German or Spanish 
+You will be eligible for the IRP. 
 
-Your chosen teacher training provider will be able to tell you whether your university and degree meet the standard and subject criteria for the international relocation payment.  
+If you are in any doubt about meeting the 3-month requirement, please email us for advice before planning your move. 
 
-### 3. Type of visa for trainees
-
-You must come to England on one of the visas listed here. Most trainees will need a Student visa. Most salaried trainees will need a Skilled Worker visa. 
-
-For the following visas, you must have applied for your visa from outside the UK:  
-
-* [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa)
-
-* [Student visa](https://www.gov.uk/student-visa)  
-
-* [Youth Mobility Scheme](https://www.gov.uk/youth-mobility)  
-
-* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa)
-
-* [Family visa](https://www.gov.uk/uk-family-visa)
-
-* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  
-
-* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa)
-
-* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  
-
-For the following Ukrainian and Afghan visas, you can apply for your visa from within or from outside the UK:  
-
-* [Afghan citizens resettlement scheme](https://www.gov.uk/guidance/afghan-citizens-resettlement-scheme)
-
-* [Afghan Relocations and Assistance Policy](https://www.gov.uk/government/publications/afghan-relocations-and-assistance-policy/afghan-relocations-and-assistance-policy-information-and-guidance)
-
-* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa)
-
-* [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
-
-### 4. Date of arrival for trainees
-
-You must move to the UK no more than 3 months before the start of your course.   
-
-### Non-eligibility for trainee teachers
+### Non-eligibility
```

Comment:
@laurahermionetennant92 is it worth adding in something about you're not eligible if you're training to teach you must already be a teacher?

Reply:
Will decide when trainee content agreed on policy side